Feature/us6550 utility classes

zarahzachz opened the pull request:
@tcmacdonald I cleaned up a lot of redundant code, using Sass mixins to accomplish a lot of the "mobile-specific classes".

Corresponds with crdschurch/crds-styleguide#49

tcmacdonald commented on assets/stylesheets/base/_utilities.scss:
```diff
@@ -398,97 +156,67 @@ a.underline,
-  }
 }
 
+@include floats;
```
You've got a couple mixins here that are only used in one place. Does that make sense? Should mixins belong here and not in the mixins module?

zarahzachz commented:
@tcmacdonald I've cleaned up the mixins, pulling out the `@mixin` wrapper on the single-use cases, and added comments explaining my thought-process on the code.

zarahzachz commented:
@tcmacdonald I added some more instruction in the Sass file on what types of classes would be rendered - giving more instruction to the developer.

Ready for review.

zarahzachz commented:
@tcmacdonald I refactored the utility classes and stripped out the Sassy logic.

Ready for review.
